app/javascript/util/listingUtil.ts

```diff
@@ -371,16 +393,23 @@ export const groupAndSortUnitsByOccupancy = (
    */
   const unitsWithOccupancy = addUnitsWithEachOccupancy(unitsCopy)
   /*
-   * We have to derive the max income using ami charts, so this mapping goes through each unit
-   * and does that and adds that max income to the unit object
+   * We have to derive the max and min income using ami charts, so this mapping goes through each unit
+   * and does that and adds that max and min income to the unit object.
+   * The Min income is derived through two means: by the AMI Chart for that value or via the BMR_Rental_Minimum_Monthly_Income_Needed field.
+   * The former is only applicable when the listing is a Section 415 housing listing (meaning there are discrete AMI ranges), while the latter
+   * is calculated in Salesforce by multiplying the base rent by the rent multiple (usually 2x).
+   * To determine this, we check if there is a BMR value and use that if there is, if not, we try to use the minimum AMI field.
+   * If neither of those are available, the value becomes -1, which forces the table to display "Up To".
```

Comment:
Long multi-line blocks of comments are a bit of a code smell. The comments themselves are not the core issue, but they indicate that the logic for creating the pricing table is overly complex and/or hard to read, and may become difficult to update or debug in the future.

With the current priorities of our engineers, we don't have the luxury of trying to refactor this code, but we should look into this later when we have time.

Reply:
For sure